department-of-veterans-affairs/next-build
# Adds Google analytics and US gov analytics info

## Description
Relates to #[16106](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16106). (or closes?)

Adds pageview hook, USA gov anayltics tag to next build
Note: Will need the following ENV files populated to test:
GOOGLE_TAG_MANAGER_AUTH=
GOOGLE_TAG_MANAGER_PREVIEW=
GOOGLE_TAG_MANAGER_ID=

We are awaiting access to those files currently


## Testing done
Local testing to verify network requests are firing for Google Tag Manager

## Screenshots
<img width="646" alt="image" src="https://github.com/department-of-veterans-affairs/next-build/assets/61624970/83fa113e-fc65-45c0-af59-765fbb4ad5c6">

<img width="1662" alt="image" src="https://github.com/department-of-veterans-affairs/next-build/assets/61624970/242ed7cf-71f1-4daf-a4c6-409530d79049">


## QA steps
Navigate to any page in next build
Verify that a GTM event fires

```[tasklist]
```

## Is this PR blocked by another PR?
- Add the `DO NOT MERGE` label
- Add links to additional PRs here:
